steepshot_io/static/js/var-helpers.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * Created by evgeni on 14.06.18.
+ */
+
+var graphPost
```

Comment:
@roman4hik what is the purpose of this file?

Reply:
@pmartynov i created this file to move all js var in this file later i changed his mind and made another, and  file  forgot deleted
i fixed  this  error